src/Makefile.am

```diff
@@ -87,6 +96,14 @@ pkglib_LTLIBRARIES =
 BUILT_SOURCES =
 CLEANFILES =
 
+# TODO: Because of the way Windows linking works, every plugin that will be
```

Comment:
This is redundant with the next sentence. I would just combine them as:
# Because Windows linking doesn't support undefined symbols, every plugin that is ported to Windows will need a line like this:
# <plugin>_la_LIBADD = daemon/libcollectd.la